CMakeLists.txt

```diff
@@ -1,18 +1,6 @@
 cmake_minimum_required(VERSION 3.7.2)
 
-set(version_regex "([0-9.]+)")
-file(STRINGS "src/h3lib/VERSION" h3_version
```

Comment:
You should also see if you can remove the `h3_version` file from the repo successfully.

Reply:
There isn't one. I'm guessing that's an older file, and we never updated this line?

Reply:
oh, sorry, I misread this line. nevermind!